scrapbook-gateway-service/routes/image.py

```diff
@@ -0,0 +1,44 @@
+import requests
+from flask import Blueprint, request, jsonify
+from service_utils import session_service 
+from config import G_DRIVE_SERVICE_URL__DEV
+import sys
+
+image_upload_api = Blueprint('image_upload_api', __name__)
+
+
+@image_upload_api.route('/image/upload/<imageid>', methods=["POST"])
+@session_service.checkUserSession()
+def uploadImage(imageid):
+    """
+    This function uploads an image
+    @params - A POST request sent to the googledrive service to upload an image
+    @return - http status code
+    """
+    try:
+        response = requests.post(f'{G_DRIVE_SERVICE_URL__DEV}/image/upload/{imageid}', headers = request.headers, data = request.data)
```

Comment:
What is `imageid` here? You're gonna be uploading the image. This should be the albums GoogeDriveID